postgres/init/90-public-efhistory.sql

```diff
@@ -60,6 +60,7 @@ COPY public.__efmigrations_history_portal (migration_id, product_version) FROM s
 20220916132329_CPLP-1406-AddConsentAssignedOfferSubscription	6.0.4
 20220920100432_CPLP-831-AddConsentAssignedOffer	6.0.4
 20220921112615_CPLP-1409-ExpandNotificationTypes	6.0.4
+20220922092106_CPLP-1422-AddDocumentToConnector	6.0.4
```

Comment:
this is in conflict with https://github.com/tractusx-team-portal-onboarding/devenv/pull/137, the migration needs to be recreated and re-applied

Reply:
updated